cerebrium/prebuilt-models/other-models/salesforce-codegen.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: "Salesforce Codegen"
+description: "CodeGen is a family of autoregressive language models for program synthesis"
+---
+
+CodeGen is a family of autoregressive language models for program synthesis. Mono models were trained on Python code and multi was trained on multiple programming languages such as C, C++, Go, Java, JS and Python.
+It is best used giving it a text prompt describing the code you would like it to generate - you can read more [here](https://huggingface.co/Salesforce/codegen-350M-multi). 
```

Comment:
The tense is a bit strange.

Maybe: The best way to use CodeGen is to give it a prompt...